tests/ui/simd/intrinsic/generic-elements-pass.rs

```diff
@@ -19,9 +19,14 @@ struct i32x8([i32; 8]);
 #[rustc_intrinsic]
 unsafe fn simd_insert<T, E>(x: T, idx: u32, y: E) -> T;
 
+#[rustc_intrinsic]
+unsafe fn simd_insert_dyn<T, E>(x: T, idx: u32, y: E) -> T;
```

Comment:
well, what is happening is that because this definition here has no body, this definition is marked as `must_be_overridden`. 

https://github.com/rust-lang/rust/blob/master/compiler/rustc_middle/src/ty/util.rs#L1758

So, I think you just can't redefine intrinsics with a fallback implementation in in tests like this with or without a body and have it all hook up right now? There is no mechanism that makes the symbol in this module have the implementation from core.

I _think_ that is therefore only a problem for tests, and I can just import the correct implementation from std in this test (`use std::intrinsics::simd::simd_extract_dyn`). Is that sufficient, or should we try to make this `rustc_intrinsic` definition work?

Reply:
btw if I do give the intrinsic a body here that is just `unreachable!()`, that will panic. Again, there is no logic that replaces the body here with the fallback from `core`.

Reply:
Yes that would be a great cleanup. :)  But indeed it should be a separate PR.